src/entities/crm/crm-user.ts

```diff
@@ -69,3 +69,7 @@ export interface AssociateUsersPayload {
   readonly externalUserReferenceId: string
   readonly staffId: string
 }
+
+export interface PersonPayload {
+  person: string
```

Comment:
nit: not really necessary, you could also just take a string parameter directly.

Reply:
done